tests/integration/test_charm.py

```diff
@@ -147,6 +156,23 @@ async def test_scale_up(ops_test: OpsTest) -> None:
     )
 
 
+async def test_login(
+    ops_test: OpsTest,
+    page: Page,
+    ext_idp_service: ExternalIdpService,
+    public_traefik_app_name: str,
+) -> None:
+    url = await get_reverse_proxy_app_url(ops_test, public_traefik_app_name, APP_NAME)
+    redirect_uri = os.path.join(url, OAUTH_CALLBACK_PATH)
+
+    await page.goto(os.path.join(url, "api/v0/auth"))
+
+    await complete_auth_code_login(page, ops_test, ext_idp_service)
+
+    # TODO: Add response and session validation
+    await page.wait_for_url(redirect_uri + "?*")
```

Comment:
Currently the tests pass, but this does not work. The reason is that the state/nonce cookies are not there when the user is redirected to the callback because of the [cookies' path](https://github.com/canonical/identity-platform-admin-ui/blob/main/pkg/authentication/cookies.go#L14)